app/assets/stylesheets/components/_print-link.scss

```diff
@@ -3,8 +3,9 @@
   margin-bottom: 2em;
 
   .app-c-print-link__link {
-    background: image-url("icon-print.png") no-repeat 0 50%;
-    padding: 0.5em 0 0.5em 28px;
+    background: image-url("icon-print.png") no-repeat 10px 50%;
+    padding: 0.5em 0.5em 0.5em 38px;
+    margin-left: -10px;
```

Comment:
What's the reason for this change? It seems to be making the code more complicated.

Reply:
It was to give breathing space to the focus stage on the print icon and bring it in line with the next/previous buttons it sometimes sits by. This is the current state:

![Screen Shot 2019-09-04 at 13 57 04](https://user-images.githubusercontent.com/31649453/64256781-04ea2b00-cf1c-11e9-87de-3a06e579973a.png)